pom.xml

```diff
@@ -7,6 +7,17 @@
     <groupId>8thlight</groupId>
```

Comment:
I would change this `groupId` to `me.karakelley` to match your package structure.